assets/javascripts/discourse/components/assigned-topic-list-item.hbs

```diff
@@ -0,0 +1,88 @@
+{{!
+  The `~` syntax strip spaces between the elements, making it produce
+  `<a class=topic-post-badges>Some text</a><span class=topic-post-badges>`,
+  with no space between them.
+  This causes the topic-post-badge to be considered the same word as "text"
+  at the end of the link, preventing it from line wrapping onto its own line.
+}}
+{{#if this.bulkSelectEnabled}}
+  <td class="bulk-select topic-list-data">
+    <input type="checkbox" class="bulk-select" />
+  </td>
+{{/if}}
+<td class="main-link clearfix topic-list-data" colspan="1">
+  <span class="link-top-line">
+    {{~raw "topic-status" topic=this.topic}}
+    {{~#if this.isPrivateMessage}}
+      {{~d-icon "envelope" class="private-message-icon"}}
+    {{~/if}}
+    {{~topic-link this.topic class="raw-link raw-topic-link"}}
+    {{~#if this.topic.featured_link}}
+      {{~topic-featured-link this.topic}}
+    {{~/if}}
+    {{~#if this.showTopicPostBadges}}
+      {{~raw
+        "topic-post-badges"
+        unread=this.topic.unread
+        unseen=this.topic.unseen
+        url=this.topic.lastUnreadUrl
+        newDotText=this.newDotText
+      }}
+    {{~/if}}
+  </span>
+  <div class="link-bottom-line">
+    {{#unless this.hideCategory}}
+      {{#unless this.topic.isPinnedUncategorized}}
+        {{category-link this.topic.category}}
+      {{/unless}}
+    {{/unless}}
```

Comment:
```suggestion
    {{#unless (and this.hideCategory this.topic.isPinnedUncategorized)}}
        {{category-link this.topic.category}}
    {{/unless}}
```

Reply:
hah, in the end linter did force me to change it to `if` 😛